lib/generators/blacklight/blacklight_access_controls.rb

```diff
@@ -0,0 +1,19 @@
+Blacklight::AccessControls.configure do |config|
+  # This specifies the solr field names of permissions-related fields.
+  # The default fields used are shown below, if you index your permissions to other fields update the configuration below.
+  # If you change these, you must also update the permissions request handler in your solrconfig.xml to return those values
+  #
+  # config.permissions.discover.group       = Solrizer.solr_name("discover_access_group", :symbol)
+  # config.permissions.discover.individual  = Solrizer.solr_name("discover_access_person", :symbol)
+  # config.permissions.read.group           = Solrizer.solr_name("read_access_group", :symbol)
+  # config.permissions.read.individual      = Solrizer.solr_name("read_access_person", :symbol)
+  # config.permissions.download.group           = Solrizer.solr_name("download_access_group", :symbol)
+  # config.permissions.download.individual      = Solrizer.solr_name("dowload_access_person", :symbol)
```

Comment:
I think my previous comment got lost in an update or a rebase, so I'll repeat them here.  The Config class in blacklight-access_controls doesn't have the same wrappers and hash accessors that hydra-access-controls Config does, so instead of calling all those chained methods, you can just set the variable directly.

So, instead of 
```
config.permissions.discover.group = Solrizer.solr_name("discover_access_group", :symbol)
```
You can just write
```
config.discover_group_field = Solrizer.solr_name("discover_access_group", :symbol)
```

So the initializer should be something like this:
```
Blacklight::AccessControls.configure do |config|
  # This specifies the solr field names of permissions-related fields.
  # The default fields used are shown below, if you index your permissions to other fields update the configuration below.
  # If you change these, you must also update the permissions request handler in your solrconfig.xml to return those values
  #
  # config.discover_group_field = Solrizer.solr_name("discover_access_group", :symbol)
  # config.discover_user_field  = Solrizer.solr_name("discover_access_person", :symbol)
  # config.read_group_field     = Solrizer.solr_name("read_access_group", :symbol)
  # config.read_user_field      = Solrizer.solr_name("read_access_person", :symbol)
  # config.download_group_field = Solrizer.solr_name("download_access_group", :symbol)
  # config.download_user_field  = Solrizer.solr_name("dowload_access_person", :symbol)
  #
  # Specify the user model
  # config.user_model = 'User'
end
```